app/controllers/links_controller.rb

```diff
@@ -38,6 +40,19 @@ def documentation_url(section = "", options = {})
     root_url + (section || '')
   end
 
+  # For new documentation at docs.theforeman.org
+  # options:
+  #   :section
+  #   :index    'foreman-el' (default), 'foreman-deb' or 'katello'
+  #   :chapter
+  def new_documentation_url(options)
+    version = SETTINGS[:version].short
+    index = options[:index] || 'foreman-el'
```

Comment:
I wonder if we can somehow detect this automatically based on the host.

Reply:
Hmm, not sure how to check it properly TBH

* foreman-el - check somehow presence of `rpm` packages?
* satellite - check for presence of satellite-theme plugin
* katello - same as above
* foreman-deb - check `deb` packages?
* orcharhino - no idea

Reply:
There is also another way to go about it: we can easily identify it in the installer so it could be a setting in settings.yaml.

As for satellite: satellite-theme would probably override the method in branding and alias it to the documentation url method.

Orcharhino may take a similar approach.

This morning I was talking to @ares that maybe we should aim for what Katello did and try to get the new docs as the default docs for Foreman 3.0. In that case, it is something we should resolve soon.

Reply:
I tend to agree that auto-detection of this may be a bit flawed or fragile over time. If this is based off a `setting.yaml` entry it becomes more predictable for the code itself here and gives control to the various spins.